.github/label-actions.yml

```diff
@@ -1,37 +1,50 @@
 # When `devnet-test-e2e` is added, also assign `devnet` and `push-image` to the PR.
```

Comment:
@okdas Let's make sure to document the learnings / instructions from here as well: https://github.com/pokt-network/protocol-infra/pull/18